src/server/session.ts

```diff
@@ -1537,6 +1537,11 @@ namespace ts.server {
             const position = this.getPositionInFile(args, file);
             const projects = this.getProjects(args);
 
+            const renameInfo = this.mapRenameInfo(
+                this.getRenameInfo(args), Debug.checkDefined(this.projectService.getScriptInfo(file)));

```

Comment:
- this change isn't really related to the issue, it's just a small refactor to cut some chars while we're at it
- i think `this.getRenameInfo(args)` is the same as `defaultProject.getLanguageService().getRenameInfo(...args)` because it works, but now there is no mention of using the default project anymore and i'm not sure if this change is correct.

Reply:
I think these are probably the same, but experience would advise not touching it if it’s not broken 😅